registry.json

```diff
@@ -230,6 +230,30 @@
     "listed": true,
     "version": "2.1.0"
   },
+  "Serilog": {
+    "listed": true,
+    "version": "2.8.0"
+  }
```

Comment:
missing `,` after each entry.

Reply:
Ah, sorry about that. That should be fixed now.